docs/distributing-policies/publish-policy-to-artifact-hub.md

```diff
@@ -0,0 +1,109 @@
+host your policies.
+:::
+
+## Prepare your Git repository
+
+Artifact Hub crawls Git repositories looking for special metadata files.
+
+There are different kind of layouts the Git repository can have. They
+are all documented in depth in the [official Artifact Hub documentation](https://artifacthub.io/docs/topics/repositories/#kubewarden-policies-repositories).
+
+Artifact Hub is pretty flexible and allows you to organize your code in these ways:
+
+* Have a Git repository dedicated to Artifact Hub: this repository will not contain
+  any policy source code. It will be a collection of the YAML files required by
+  Artifact Hub.
+* Add a Artifact Hub directory inside of the Git repository that holds the source
+  of your policy. This is an iteration of the previous approach, the only difference
+  is that it focuses just on one policy: the one defined inside of the Git repository.
+  This approach allows to keep multiple versions of the policy published on Artifact Hub.
+* Add the `artifacthub-pkg.yml` and the `artifacthub-repo.yml` files to the root
+  of the Git repository that holds the policy source code. This approach is the 
+  simplest one. The only limitation is that only the latest version of the policy
+  will be visible on Artifact Hub.
+
+The last approach is what we used inside of our official policy templates.
+The Git repository that is scaffolded includes the `artifacthub-pkg.yml`
+and `artifacthub-repo.yml` files.
+
+## Publishing Steps
+
+Before publishing a policy to Artifact Hub, you must create an account on
+the [website](https://artifacthub.io/).
+
+Before publishing the policy, ensure your Git repository has the proper layout.
+
+If you created the policy using one of our templates, ensure the `artifacthub-pkg.yml` file
+has correct data inside of it.
+The format of the `artifacthub-pkg.yml` is described [here](https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-pkg.yml).
+
+Finally, ensure your policy is published inside of a container registry or on a
+web server.
+
+:::note
+Right now the contents of the `artifacthub-repo.yml` file are not relevant.
+:::
+
+Once everything is in place, log into Artifact Hub and go to your
+[control plane](https://artifacthub.io/control-panel/repositories?page=1).
+
+Decide whether you want to publish the policy as a user or under an Artifact Hub
+organization you belong to. This is done by choosing the correct *"control panel context"*.
+
+Then press the *"Add"* button and fill the form:
+
+  * Choose *"Kubewarden policies"* as kind.
+  * Enter a *"Name"* and *"Display name"* of your choice.
+  * Enter the URL to your Git repository.
+
+Finally, press the *"Add"* button. This will bring you back to the *"Repositories"*
+page, where you will see your freshly created repository.
+
+Each repository has several information fields. Find the *"ID"* property of the
+repository you just created and copy it.
```

Comment:
@divya-mohan0209 : thought about what you said during the standup. What about making everything less "artifact hub docs":

```suggestion
From your control plane, add a new Artifact Hub by following the guided procedure.
Once this done, copy the value of the *"ID"* property that Artifact Hub
assigned to the freshly created repository.
```

Reply:
I think this looks good, Flavio. However, if the guided procedure doesn't include the actual steps you mentioned Id rather we retain it as-is.

Reply:
Currently ArtifactHub's docs do not have this kind of content. Hence I'll leave this text unchanged.

Reply:
That is fine by me!